docker_custodian/__about__.py

```diff
@@ -1,4 +1,4 @@
 # -*- coding: utf8 -*-
 
-__version_info__ = (0, 6, 0)
```

Comment:
This feels more like a new feature than a bug fix, so I'd probably make this `0, 7, 0`